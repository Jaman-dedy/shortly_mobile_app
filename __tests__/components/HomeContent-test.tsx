import React from 'react';
import renderer from 'react-test-renderer';
import {cleanup} from '@testing-library/react-native';

import HomeContentComponent from '../../src/components/HomeContent';

import MockThemes from '../../mock/MockTheme';

afterEach(cleanup);
test('renders correctly', async () => {
  const tree = await renderer
    .create(
      <MockThemes>
        <HomeContentComponent />
      </MockThemes>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
