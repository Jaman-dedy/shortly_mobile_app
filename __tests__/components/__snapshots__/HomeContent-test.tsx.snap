// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#F0F1F6",
        "flex": 3,
      },
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#35323E",
          "fontFamily": "Poppins-Medium",
          "fontSize": 34,
          "fontWeight": "bold",
          "paddingTop": 22,
          "textAlign": "center",
        },
      ]
    }
  >
    Shortly
  </Text>
  <View
    style={
      Array [
        Object {},
      ]
    }
  >
    < />
    <View
      style={
        Array [
          Object {
            "paddingBottom": 8,
          },
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#35323E",
              "fontFamily": "Poppins-Medium",
              "fontSize": 22,
              "fontWeight": "bold",
              "textAlign": "center",
            },
          ]
        }
      >
        Let's get started
      </Text>
      <Text
        style={
          Array [
            Object {
              "color": "#35323E",
              "fontFamily": "Poppins-Medium",
              "fontSize": 18,
              "textAlign": "center",
            },
          ]
        }
      >
        Paste your first link into
the field to shorten it
      </Text>
    </View>
  </View>
</View>
`;
